app/src/main/java/net/gsantner/opoc/preference/GsPreferenceFragmentCompat.java

```diff
@@ -152,6 +153,12 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
         addPreferencesFromResource(getPreferenceResourceForInflation());
 
 
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.KITKAT) {
```

Comment:
Please don't make any changes here. The point where you can do this is SettingsActivity, and it's only one line required:  In doUpdatePreferences.

    setPreferenceVisible(R.string.pref_key__X, Build.VERSION.SDK_INT >= Build.VERSION_CODES.X);

Reply:
Great to learn this.